content/bootloader.md

```diff
@@ -17,13 +17,13 @@ section: software-troubleshooting
 tableOfContents: true
 ---
 
-Systemd-boot is the boot-loader for Pop!_OS 18.04 and above while GRUB is the boot-loader for Ubuntu and Pop!_OS when installed in Legacy BIOS mode. A boot-loader takes care of getting the operating system started up. It is also responsible for allowing the user to select between multiple operating systems at boot. Sometimes, GRUB or systemd-boot can break, and it may not let you boot into your computer to fix the problem.
+Systemd-boot is the bootloader for Pop!_OS 18.04 and above while GRUB is the bootloader for Ubuntu and Pop!_OS when installed in Legacy BIOS mode. A bootloader takes care of getting the operating system started up. It is also responsible for allowing the user to select between multiple operating systems at boot. Sometimes, GRUB or systemd-boot can break, and it may not let you boot into your computer to fix the problem.
 
 ### Important Note
 
-If you need to configure grub-pc (for example, after an update), installing grub to all devices will break GRUB. You will need to install to `/dev/sda` _not_ `/dev/sda1`.
+If you need to configure grub-pc (for example, after an update), installing GRUB to all devices will break GRUB. You will need to install to `/dev/sda` _not_ `/dev/sda1`.
 
-On a fresh install of Pop!_OS 18.04, <u>systemd-boot</u> is used rather than the <u>GRUB</u> boot-loader, and the following instructions do not apply please refer to the <u>systemd-boot</u> section on this page.
+On a fresh install of Pop!_OS 18.04 and newer, <u>systemd-boot</u> is used rather than the <u>GRUB</u> bootloader, and the following instructions do not apply please refer to the <u>systemd-boot</u> section on this page.
```

Comment:
This is a run-on sentence. Needs to be something like "the following instructions do not apply; please refer to" etc, or two separate sentences.

This is also one of the pages that could _really_ benefit from having some sections collapsed by default.

Reply:
I believe I addressed this concern with my update